File: Araba/tofas.java
public class tofas extends araba implements IKlima{
    public tofas(double var1, double var2){
        super(var1, var2);
    }

    public void power(){
        this.acceleration = 5;
    }

    public void open(){
        this.acceleration *= 0.9;
    }
}

File: Araba/ArabaCsharp/Tofas.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace YTM
{
    public class Tofas : Araba, IKlima
    {
        public Tofas(double var1, double var2) : base(var1, var2){}

        public override void Power(){
            this.acceleration = 5;
        }

        public void Open(){
            this.acceleration *= 0.9;
        }
    }
}